frontend/src/app/middleware/middleware.service.ts

```diff
@@ -80,20 +80,29 @@ export class MiddlewareService {
       .post(`${API_BASE}/claims`, claim, {
         headers: this.headers,
       })
+      .subscribe( 
+        data => {},
+        error => {},
```

Comment:
Can't this just be `undefined` here to avoid the function call since you're not using it?